### PR TITLE
call update_permissions_from_mc command on startup

### LINKED ISSUE #4882: Pre-load webext permissions on deploy/start
#4540 added a command that pulls and updates descriptions of supported webextention permissions from mozilla-central.  We should automate this so it's called every time addons-server is deployed or started.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #4882 ?
Did this because it was super simple... but not so sure it's a good idea (especially for local development where olympia is restarted a lot).

### CODE REVIEW

**Comment by diox** [comment]
Not sure if it's the best place to do it. I'd rather have it done on deploy only (we might restart instances, or start more than one). Maybe in `Dockerfile.deploy` then ?

**Comment by diox** [comment]
Alternatively: in a cron once a week or more ?

**Comment by eviljeff** [comment]
`Dockerfile.deploy` works for prod/stage/dev (I guess).  `Dockerfile` is used for local development but I assume only called when you create the container(?) - stick something in `make update_docker` or something?

**Comment by diox** [comment]
Yeah. You'd probably need to stick it in both `Dockerfile.deploy` (for dev/stage/prod) and some `make` command (for our local envs)

**Comment by eviljeff** [comment]
https://github.com/mozilla/addons-server/pull/4884/commits/39224ecf8bb4473ea6b8f619c7740adca23d505a adds to `Dockerfile.deploy`, `make update_docker`, `make initialize_docker` (I'm assuming `make initialize_docker` isn't a duplicate with `Dockerfile.deploy`)

I tested `make update_docker` but wasn't sure how to test `Dockerfile.deploy` exactly.

**Comment by eviljeff** [comment]
There's a circleci fail (a genuine fail for a change 😱) indicating that `Dockerfile.deploy` might not be the place for a command that hits the database...

**Comment by diox** [comment]
mmm is that just because you're missing `DJANGO_SETTINGS_MODULE='settings_local'` ?

**Comment by eviljeff** [comment]
I read the lines above as only needing `DJANGO_SETTINGS_MODULE='settings_local'` for the assets related settings in https://github.com/eviljeff/olympia/blob/39224ecf8bb4473ea6b8f619c7740adca23d505a/Dockerfile.deploy#L55:L61 - I figured some other sensible default settings module would be used instead.

**Comment by diox** [comment]
mmm yeah. There should be a different `settings_local.py` file after deployment, that just does `from olympia.conf.*.settings import *` but it's not created by `Dockerfile.deploy` so I don't know...

File: Dockerfile.deploy [hunk 1]
```diff
@@ -67,4 +67,7 @@ RUN npm install \
     && DJANGO_SETTINGS_MODULE='settings_local' python manage.py compress_assets -t \
     && DJANGO_SETTINGS_MODULE='settings_local' python manage.py collectstatic --noinput
 
+# fetch updated webext permission descriptions from mozilla-central
+RUN DJANGO_SETTINGS_MODULE='settings_local' python manage.py update_permissions_from_mc
```
Comment (bqbn) [inline Dockerfile.deploy]: So what does this line do to the image?

**Comment by eviljeff** [comment]
okay, it's just the `make` changes now.

**Pull request merged**